### PR TITLE
Fix/v2 split range

### PR DESCRIPTION
**Pull request by carpawell**

### CODE REVIEW

File: pkg/services/object/get/assembly_v2.go [hunk 1]
```diff
@@ -116,7 +116,7 @@ func (exec *execCtx) rangeFromLink(link objectSDK.Link) bool {
 				rngPerChild.SetOffset(uint64(firstOffset))
 			}
 			if i == last {
-				rngPerChild.SetLength(uint64(child.ObjectSize()) - uint64(lastBound+firstOffset))
+				rngPerChild.SetLength(uint64(lastBound - firstOffset))
```
Comment (roman-khimov) [inline pkg/services/object/get/assembly_v2.go:119]: Isn't it just `uint64(lastBound)`?

Comment (carpawell) [reply pkg/services/object/get/assembly_v2.go:119]: we may range only part of the first object, then we do not need its offset

Comment (roman-khimov) [reply pkg/services/object/get/assembly_v2.go:119]: But what if we have multiple objects? You get `X:$END` of `first`, OK, then you go over some set of complete objects and then you need `0:Y`, while you do `0:Y-X`. Or not?

Comment (carpawell) [reply pkg/services/object/get/assembly_v2.go:119]: hm, seems like you are right but why do tests work then? we never range over the bounds in the tests?

**Review by roman-khimov — APPROVED** [review]

**Review by cthulhu-rider — COMMENTED** [review]
> Seems like a hotfix after some review or smth.

i'd rather formulate essence of the problem instead of obvious abstract things

**Comment by carpawell** [comment]
> i'd rather formulate essence of the problem

I have nothing to add. This line should have the exact number, it did not. The number it was had nothing about the required range, I do remember that i fixed something there but have no idea what that meant. OK, dropped the body.

**Review by cthulhu-rider — APPROVED** [review]

**Pull request merged**